### PR TITLE
Contact profile

### PR DESCRIPTION
**Pull request by tulimaki**

### CODE REVIEW

File: shuup/front/apps/customer_information/forms.py [hunk 1]
```diff
@@ -31,14 +32,20 @@ class PersonContactForm(forms.ModelForm):
...
+        )
 
     def __init__(self, *args, **kwargs):
         super(PersonContactForm, self).__init__(*args, **kwargs)
         for field in ("first_name", "last_name", "email"):
             self.fields[field].required = True
         self.initial["language"] = self.instance.language
 
+        self.fields["picture"].widget = PictureDnDUploaderWidget(
+            upload_path="/contact_pictures", kind="images", clearable=True)
```
Comment (chessbr) [inline shuup/front/apps/customer_information/forms.py]: maybe just `/Contacts`

File: shuup/front/apps/customer_information/forms.py [hunk 2]
```diff
@@ -52,10 +61,16 @@ def save(self, commit=True):
...
-        fields = ("name", "tax_number", "phone", "email", "marketing_permission")
+        fields = (
+            "name", "tax_number", "phone", "email", "marketing_permission", "picture"
+        )
 
     def __init__(self, *args, **kwargs):
         super(CompanyContactForm, self).__init__(*args, **kwargs)
+
+        self.fields["picture"].widget = PictureDnDUploaderWidget(
+            upload_path="/contact_pictures", kind="images", clearable=True)
```
Comment (chessbr) [inline shuup/front/apps/customer_information/forms.py]: maybe `/contacts` here too

File: shuup/front/apps/customer_information/settings.py [hunk 3]
```diff
@@ -13,3 +13,6 @@
 
 #: The default address book edit form
 SHUUP_ADDRESS_BOOK_EDIT_FORM = "shuup.front.apps.customer_information.forms:AddressBookFormGroup"
+
+#: Allow customers to upload profile picture
+SHUUP_COMPANY_INFORMATION_ALLOW_PICTURE_UPLOAD = False
```
Comment (chessbr) [inline shuup/front/apps/customer_information/settings.py]: customers or companies? typo?

File: shuup/front/apps/customer_information/forms.py [hunk 4]
```diff
@@ -52,10 +61,16 @@ def save(self, commit=True):
...
         model = CompanyContact
-        fields = ("name", "tax_number", "phone", "email", "marketing_permission")
+        fields = (
+            "name", "tax_number", "phone", "email", "marketing_permission", "picture"
+        )
 
     def __init__(self, *args, **kwargs):
         super(CompanyContactForm, self).__init__(*args, **kwargs)
+
+        self.fields["picture"].widget = PictureDnDUploaderWidget(
```
Comment (chessbr) [inline shuup/front/apps/customer_information/forms.py]: why here you won't check for `SHUUP_COMPANY_INFORMATION_ALLOW_PICTURE_UPLOAD` ?

File: shuup/front/settings.py [hunk 5]
```diff
@@ -116,3 +116,9 @@
 #: }
 #:
 SHUUP_FRONT_OVERRIDE_SORTS_AND_FILTERS_LABELS_LOGIC = {}
+
+
+#: Maximum allowed file size for uploads when posting to
+#: `shuup.front.views.upload.media_upload`
+#:
+SHUUP_FRONT_MAX_UPLOAD_SIZE = 500000
```
Comment (chessbr) [inline shuup/front/settings.py]: oh, this could be something around 10MB, 500MB is too much I guess 😛

Comment (tulimaki) [reply shuup/front/settings.py]: I think this is 0.5MB

**Pull request merged**